plugins/rapid7_insight_agent/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: rapid7_insight_agent
 title: Rapid7 Insight Agent
 description: Using the Insight Agent plugin from InsightConnect, you can quarantine, unquarantine and monitor potentially malicious IPs, addresses, hostnames, and devices across your organization
-version: 2.0.1
+version: 2.0.2
```

Comment:
Should this still be a minor patch?
I know technically there are no new inputs or outputs, but there is new functionality.

Let me know what you think 😄